docs/guides/mssqlserver/concepts/mssqlserver.md

```diff
@@ -0,0 +1,482 @@
+
+If you want to use an existing or custom secret, please specify that when creating the MSSQLServer object using `spec.authSecret.name`. This Secret should contain superuser _username_ as `username` key and superuser _password_ as `password` key. Secrets provided by users are not managed by KubeDB, and therefore, won't be modified or garbage collected by the KubeDB operator.
+
+Example:
+
+```bash
+$ kubectl create secret generic mssqlserver-auth -n demo \
+             --from-literal=username='sa' \
+             --from-literal=password='Pa55w0rd!'
+secret/mssqlserver-auth created
+```
+
+```bash
+$ kubectl get secret -n demo  mssqlserver-auth -oyaml
+apiVersion: v1
+data:
+  password: UGE1NXcwcmQh
+  username: c2E=
+kind: Secret
+metadata:
+  creationTimestamp: "2024-10-10T06:47:06Z"
+  name: mssqlserver-auth
+  namespace: demo
+  resourceVersion: "315403"
+  uid: dafcce02-b6a2-4e65-bdd1-db6b9b6d4913
+type: Opaque
+```
+
+### spec.storageType
+
+`spec.storageType` is an optional field that specifies the type of storage to use for database. It can be either `Durable` or `Ephemeral`. The default value of this field is `Durable`. If `Ephemeral` is used then KubeDB will create MSSQLServer database using [emptyDir](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir) volume. In this case, you don't have to specify `spec.storage` field.
+
+### spec.storage
+
+If you don't set `spec.storageType:` to `Ephemeral` then `spec.storage` field is required. This field specifies the StorageClass of PVCs dynamically allocated to store data for the database. This storage spec will be passed to the PetSet created by KubeDB operator to run database pods. You can specify any StorageClass available in your cluster with appropriate resource requests.
+
+- `spec.storage.storageClassName` is the name of the StorageClass used to provision PVCs. PVCs don’t necessarily have to request a class. A PVC with its storageClassName set equal to "" is always interpreted to be requesting a PV with no class, so it can only be bound to PVs with no class (no annotation or one set equal to ""). A PVC with no storageClassName is not quite the same and is treated differently by the cluster depending on whether the DefaultStorageClass admission plugin is turned on.
+- `spec.storage.accessModes` uses the same conventions as Kubernetes PVCs when requesting storage with specific access modes.
+- `spec.storage.resources` can be used to request specific quantities of storage. This follows the same resource model used by PVCs.
+
+To learn how to configure `spec.storage`, please visit the links below:
+
+- https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims
+
+### spec.init
+
+`spec.init` is an optional section that can be used to initialize a newly created MSSQLServer database. MSSQLServer databases can be initialized from these two ways:
+
+1. Initialize from Script
+2. Initialize from Snapshot
+
```

Comment:
Give an basic example of how you can initialize db using init script as you do not have `Initialization` part yet.